druid/examples/readme.md

```diff
@@ -0,0 +1,216 @@
+```
+cargo run --example scroll
+```
+Scrolling is a great way to show more content then can be displayed on the screen at a time. This is an example showing you how to use them.
+
+## Split
+```
+cargo run --example split_demo
+```
+An example of how to split a widget in 2 in various ways. This also includes having the user drag the border!! 
+
+## Styled_text
+```
+cargo run --example styled_text
+```
+Not all text should look the same. You are able to change a lot of things, color, size, and monospace. This example shows how ot change these propperties.
+
+## Svg
+```
+cargo run --example svg
+```
+This shows you how to display an SVG as a widget.
+
+## Switches
+```
+cargo run --example switches
+```
+Switches are usefullin many ways, this example shows how to use the druid built-in ones. This includes on/off and up/down for incrementing numeric values.
+
+## Tabs
+```
+cargo run --example tabs
+```
+Tabs allow you to seperate diferent portions of the UI. This example shows you how to use them in druid. Similair to [view switcher](##View Switcher) but with with a diferent purpose.
+
+## Timers
+```
+cargo run --example timers
+```
+Timers allow you to send events to your widgets at a certain points inthe future. This example shows how to use them.
+
+## View Switcher
+```
+cargo run --example view_switcher
+```
+Very smilair to [tabs](##Tabs) but this allows you to have more controll over it. This allows you to switch out widgets on the fly.
+
+## Showcases
+
+### Calc
+```
+cargo run --example calc
+```
+
+This is a showcase of a simple calculator. There are better ways to implement the calculator logic, but it provides all the standard operations like adition devision multiplication C and CE.
+
+### Event Viewer
+```
+cargo run --example event_viewer
+```
+
+Used as a debugging tool, this prints out mouse and keyboard events as they are received by Druid.
+
+### Flex
+```
+cargo run --example flex
+```
+
+Flex shows off all the things you can do with flex elements. You can play with all the setings and it will change in real-time.
+
+### Game Of Life
+```
+cargo run --example game_of_life
+```
+
+A simple implementation of conway's game of life. You can change the evolution speed, and pauze so you can take your time making your own creations!
+
+### Image
+```
+cargo run --example image
+```
+
+Image shows off all the knobs you can turn on images. You can play with them with real time results, which you to fogure out what settings are best for you.
+
+Please note that the image is exported with some kind of interpolation. So even when you turn interpolation off/NearestNeighbor in druid, you will still see this because thats how the image actually looks.
+
+### Scroll Colors
+```
+cargo run --example scroll_colors
+```
+
+This is a showcase is scrolling through an image gradient square. The square is devided into smaller squares each with a unique color. There are other ways to to this like one big widget with an image for example.
```

Comment:
divided

Reply:
(Actually, I'm not sure what the purpose of this example is, given that we have scroll?)

Reply:
yeah it is like the most basic showcase out there, not really doing much imho.
I propose to remove this in another PR?